torch/csrc/distributed/rpc/rpc_agent.h

```diff
@@ -93,6 +96,11 @@ class TORCH_API RpcAgent {
 
   virtual const WorkerInfo& getWorkerInfo(worker_id_t id) const = 0;
 
+  // Retrieve the timeout for all RPCs.
+  virtual const std::chrono::milliseconds& getRpcTimeout() const {
```

Comment:
Why this needs to be virtual. In which situation a derived class would like to override this value?

Can we make it inline?

Reply:
Removing it.

Marking it as inline.

Turned out pybind11 does not require it to be declared in sub-classes. pybind11 is able to look up this member function in the base class.